app/tasks/job_creator.rb

```diff
@@ -12,9 +12,9 @@ def call
     return unless processable
 
     process
-    # rescue StandardError => e
```

Comment:
Add comments to file (also CompanyCreator)

Reply:
Do either of you know if there's a way to make rescue blocks operate only in production? It's good to handle certain errors that we can anticipate in production in a way that doesn't break things, but it makes it a lot harder to find bugs in testing or development. I've just been commenting the rescue block on each branch I make and then trying to remember to uncomment it before I push, lol...